sympy/stats/joint_rv.py

```diff
@@ -233,9 +234,22 @@ def _sample_pymc3(cls, dist, size, seed):
         if dist.__class__.__name__ not in dist_list:
             return None
 
-        with pymc3.Model():
-            pymc3_rv_map[dist.__class__.__name__](dist)
-            samples = pymc3.sample(size, chains=1, progressbar=False, random_seed=seed)[:]['X']
+        #size accepts only int in case of pymc3
+        #see https://github.com/sympy/sympy/pull/21590#discussion_r649783860
+
+        draws = list(size)[1:]
+        if draws == []:
+            draws=1
+        elif len(draws) == 1:
+            draws = draws[0]
+        else:
+            raise TypeError("Invalid value for `size` in pymc3. Must be int")
+        import logging
+        logging.getLogger("pymc3").setLevel(logging.ERROR)
+        with ignore_warnings(UserWarning):
+            with pymc3.Model():
```

Comment:
Is there a way to tell pymc3 not to emit the warning rather than using `ignore_warnings` here?

Reply:
I guess there's a way. If we set compute_convergence_checks=False, then it will not emit the UserWarning regarding insufficient no of samples for checks. `ignore_warnings` is used to achieve the same effect.

Reply:
It would be better to use that rather than `ignore_warnings` because it is still useful to be able to see warnings even if there are some that we know we want to disable.

Reply:
Yes, I understand.